pom.xml

```diff
@@ -25,7 +25,7 @@
 
 		<!-- Dependency Versions -->
 		<!-- Make sure that the following block of dependencies match the version used in jasper reports found at https://github.com/TIBCOSoftware/jasperreports/blob/master/jasperreports/pom-common.xml -->
```

Comment:
That URL does not exist anymore. I found this as the new url. 
https://raw.githubusercontent.com/TIBCOSoftware/jasperreports/master/pom-parent.xml